detect_secrets_server/core/usage/scan.py

```diff
@@ -30,6 +31,14 @@ def add_arguments(self):
                 'saving state.'
             ),
         )
+        self.parser.add_argument(
+            '--always-update-state',
+            action='store_true',
+            help=(
+                'Always update the internal tracking state after a successful scan, '
+                'despite finding secrets.'
```

Comment:
@KevinHock : let me know if this description makes sense. Hoping to get a sanity check to hopefully have better descriptions for users who don't fully understand the architecture of the system.

Reply:
@domanchi I think we can put in parenthesis what we mean by that
`the internal tracking state after` ->
`the internal tracking state (latest sha hash scanned) after`